cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -372,6 +384,32 @@ func (uc *UpstreamController) updateNodeStatus(stop chan struct{}) {
 			}
 
 			switch msg.GetOperation() {
+			case model.InsertOperation:
+				klog.Infof("Received register node request, creating node")
+				if _, err := uc.createNode(name); err != nil {
+					klog.Errorf("Failed to create node: %s with error: %s", name, err)
+					continue
```

Comment:
we have retried in edgepart, continue is OK here :)

Reply:
@fisherxu continue is continue to the next for loop

Reply:
yes, I mean OK :)